File: tests/fixtures/cs/input.cs
using System;

namespace Geometry
{
    public abstract class Shape : IComparable
    {
        public const double Pi = 3.14159;
        private double _area;

        public Shape(double area)
        {
            _area = area;
        }

        public abstract double Area();

        protected static int Count() => 0;

        public string Name { get; set; }
    }

    public interface IDrawable
    {
        void Draw();
    }

    public enum Color
    {
        Red,
        Green,
        Blue
    }

    public struct Point
    {
        public int X;
        public int Y;
    }
}

File: tests/fixtures/java/input.java
package com.example.geometry;

import java.util.List;

/** A shape. */
public abstract class Shape implements Comparable<Shape> {
    public static final double PI = 3.14159;
    private double area;

    public Shape(double area) {
        this.area = area;
    }

    public abstract double area();

    protected static int count() {
        return 0;
    }
}

interface Drawable {
    void draw();
}

enum Color {
    RED,
    GREEN,
    BLUE
}
